Bottom: 04a74d84954cc67eb428c6db07846c7da2416ab5
Top:    25ce7dde3c6db06bd20a897c26f57761e279d971
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-19 20:34:10 +0800

Refresh of vhost_net-split-out-datacopy

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 5c3b9ef..c81e4e4 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -549,7 +549,8 @@ static void handle_tx_copy(struct vhost_net *net, struct socket *sock)
 		if (head == vq->num) {
 			if (unlikely(busyloop_intr)) {
 				vhost_poll_queue(&vq->poll);
-			} else if (unlikely(vhost_enable_notify(&net->dev, vq))) {
+			} else if (unlikely(vhost_enable_notify(&net->dev,
+								vq))) {
 				vhost_disable_notify(&net->dev, vq);
 				continue;
 			}
